blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -418,11 +418,11 @@ def _update_power_levels(self, tree_iter: Gtk.TreeIter, device: Device, cinfo: c
             except ConnInfoReadError:
                 bars["rssi"] = 50
             try:
-                bars["lq"] = max(50 + float(cinfo.get_lq()) / 127 * 50, 10)
+                bars["lq"] = max(float(cinfo.get_lq()) / 255 * 100, 10)
             except ConnInfoReadError:
                 bars["lq"] = 50
```

Comment:
Should this be changed to 0 (and the corresponding line in the `tpl` except to `0`)?

Reply:
Good catch. Mostly right, except that 0 is invalid in any case and it needs to be 10. :see_no_evil: